### ISSUE TITLE
Fix the module/updateinfo.feature tests

### ISSUE THREAD

**Issue by jan-kolarik** [rpm-software-management/ci-dnf-stack#1548]
Quoted from https://github.com/rpm-software-management/ci-dnf-stack/pull/1547:

> After merging https://github.com/rpm-software-management/ci-dnf-stack/pull/1540, it was revealed that this test in `module/updateinfo.feature` wasn't actually testing anything, and the behavior it intends to test is (and has long been?) broken. The test was expecting an empty stdout (or `<REPOSYNC>`), which it was getting, but only after refactoring the tests did we notice the message on stderr:
> 
> ```
> Unknown argument "--list" for command "updateinfo".
> ```
> 
> For now, I've updated the test to call the correct command, `updateinfo list`, and disabled the test since it is now failing (but in a more valid way):
> 
> ```
> Assertion Failed: Stdout is not empty, it contains:
> Name                   Type        Severity                  Package              Issued
> FEDORA-2019-0329090518 enhancement none     nodejs-1:8.14.0-1.x86_64 2019-03-29 00:00:00
> ```
> 
> Another option would be to fix the test to use the correct command, but leave it enabled and failing.
> 
> cc @pkratoch, do you have an idea of what the problem is and how long it might take to fix? If it's complicated/low priority, we can go ahead and disable the test and put an issue on the backlog for it, otherwise maybe it's better to go ahead and fix it instead of disabling then immediately re-enabling it.

**Comment by kontura** [comment]
I believe it problem is here: https://github.com/rpm-software-management/dnf5/blob/main/libdnf5/advisory/advisory_collection.cpp#L74-L77 it wasn't possible to implement the applicability check when I was working on it.

However now we module support so we could resolve this.
